app/pages/sections/catalogue-solutions/order-item/controller.js

```diff
@@ -206,17 +206,76 @@ export const postSolutionOrderItem = async ({
     selectedPrice,
     formData,
   });
+
+  await postData({
+    endpoint, body, accessToken, logger,
+  });
+  logger.info(`Order item successfully created for order id: ${orderId}`);
```

Comment:
Might be useful to provide a little context as to what order item was created e,g. CatalogueItemName? This is obviously very minor suggestion, so doesn't need to go into this PR.